truffleruby/src/main/java/org/truffleruby/core/CoreLibrary.java

```diff
@@ -1603,6 +1603,8 @@ public Object getTruffleKernelModule() {
             "/core/truffle/support.rb",
             "/core/truffle/boot.rb",
             "/core/truffle/debug.rb",
+            "/core/truffle/string_ops.rb",
```

Comment:
I don't really care that we've added a new class, but we should consolidate `Truffle::StringOps` and `Truffle::String` then. This also breaks a bit with the naming convention, as we have `Truffle::Array`. So, if we want to use a new convention, I'm fine with that, but we should consistently apply it.